.gitignore

```diff
@@ -6,6 +6,7 @@ spec/dummy/tmp/
 spec/dummy/public/assets/
 doc
 .yardoc
+yarn.lock
```

Comment:
do we want to ignore this file?
ref: https://github.com/yarnpkg/yarn/issues/1583

Reply:
Our applications use the official npm client, so if you're using yarn as a developer it doesnt make sense for this to be commited